src/picologging/logger.cxx

```diff
@@ -31,7 +31,7 @@ PyObject* Logger_new(PyTypeObject* type, PyObject* args, PyObject* kwds)
         Py_INCREF(self->name);
         self->parent = Py_None;
         Py_INCREF(self->parent);
-        self->propagate = true;
+        self->propagate = Py_True;
```

Comment:
The field should stay as a C++ boolean and have a custom setter instead. This way, we can do faster checking in C++ and not have to do the conversion on access.